README.md

```diff
@@ -109,7 +116,7 @@ Write this grammar using EBNF with common extensions
 
 ## A2.3
 
-«replace this with your answer»
+Please open FSM.pdf
```

Comment:
The diagram is fine past S2, but the first three states are kind of screwy.

In a state machine you have states (the circles) and directed transitions between the circles. The transitions are each labelled with an event. When you are in a particular state and you receive an event, you look for the transition with that even as a label and follow it to the next state.

You can't have two transitions from a particular state with the same label (because how could the state machine choose between them) and every transition must be caused by an event.

# -3